CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Changelog
 
 ## v6.3.1 (Not yet published)
+### Changed
 
+-   Changed default font-size of tooltip text to `12px`.
```

Comment:
-   Changed default font-size of `.mat-tooltip` text to 12px.

I think it'd be good to specify it's material's tooltip.